packages/nvidia_nat_test/src/nat/test/llm.py

```
@@ -66,14 +66,14 @@ async def async_sleep(self) -> None:
-@register_llm_provider(config_type=TestLLMConfig)
-async def test_llm_provider(config: TestLLMConfig, builder: Builder):
+@register_llm_provider(config_type=MockLLMConfig)
+async def test_llm_provider(config: MockLLMConfig, builder: Builder) -> AsyncGenerator[LLMProviderInfo, None]:
     """Register the `nat_test_llm` provider for the NAT registry."""
     yield LLMProviderInfo(config=config, description="Test LLM provider")
 
 
-@register_llm_client(config_type=TestLLMConfig, wrapper_type=LLMFrameworkEnum.LANGCHAIN)
-async def test_llm_langchain(config: TestLLMConfig, builder: Builder):
+@register_llm_client(config_type=MockLLMConfig, wrapper_type=LLMFrameworkEnum.LANGCHAIN)
+async def test_llm_langchain(config: MockLLMConfig, builder: Builder):
```

Ruff (0.14.10)

Unused function argument: `builder`

(ARG001)